oauth.js

```diff
@@ -86,7 +89,7 @@ function redirectUrl(params, oauthConfig) {
 module.exports = function(app, db) {
   // begin a new oauth log in flow
   app.get('/api/login', function(req, res) {
-    setupOAuthFlow(req, 'signin', null, function(err, params, oauthConfig) {
+    setupOAuthFlow(req, 'signin', null, null, function(err, params, oauthConfig) {
```

Comment:
If you pass `null` in the fourth position, `options` is `null` and `options.acrValue` is `TypeError`, no?

Reply:
Thanks @jrgm, updated the `setupOAuthFlow` function so we don't need to pass null anymore.